tests/unit/metrics/test_adaptive_threshold.py

```diff
@@ -18,6 +18,8 @@
     [
         (torch.Tensor([0, 0, 0, 1, 1]), torch.Tensor([2.3, 1.6, 2.6, 7.9, 3.3]), 3.3),  # standard case
         (torch.Tensor([1, 0, 0, 0]), torch.Tensor([4, 3, 2, 1]), 4),  # 100% recall for all thresholds
+        (torch.Tensor([1, 1, 1, 1]), torch.Tensor([4, 3, 2, 1]), 1),  # use minimum value when all images are anomalous
```

Comment:
Maybe we could add an explicit check in the metric implementation for this case, similar to the check you added for only normal images.

Reply:
I have just added an explicit check for this case where all images are anomalous 💪 It would be great if you could check if my change is appropriate.